src/pycram/process_modules/pr2_process_modules.py

```diff
@@ -77,8 +77,14 @@ def feedback_callback(msg):
             pass
 
         goal = Pr2GripperCommandGoal()
-        goal.command.position = 0.0 if designator.motion == GripperState.CLOSE else 0.1
-        goal.command.max_effort = 50.0
+
+        position_map = {
```

Comment:
That should be in the robot_description

Reply:
well this wasnt in the description before i can move it later there